modules/sql-engine/src/integrationTest/sql/order/test_offset.test

```diff
@@ -0,0 +1,36 @@
+# name: test/sql/order/test_offset.test
+# description: offset clause tests
+# feature: SQL F861 (Top-level <result offset clause> in <query expression>)
+# group: [order]
+
+statement ok
+PRAGMA enable_verification
+
+statement ok
+CREATE TABLE test (b INTEGER);
+
+statement ok
+INSERT INTO test VALUES (22), (2), (7);
+
+query I
+SELECT b FROM test ORDER BY b DESC OFFSET 1;
```

Comment:
should we also check syntax?

<result offset clause> ::=
  OFFSET <offset row count> { ROW | ROWS }